src/masonite/authorization/AuthorizesRequest.py

```diff
@@ -0,0 +1,5 @@
+class AuthorizesRequest:
+    def authorize(self, permission, *args):
+        from ..facades import Gate
```

Comment:
Does this need to be a lazy import? Can we import this at the top?

Reply:
Maybe yes, I will check if there are no issues while doing this